### ISSUE TITLE
integrate pyscript jupyter kernel into HA integration Jupyter Lab

### ISSUE THREAD

**Issue by SnokxStefan** [custom-components/pyscript#644]
Dear, 
There is now a standard HA integration for jupyter lab.
Is it possible to install and use the pyscript jupyter kernel from here ?
i run HA on a raspberry.

Thanks for helping an non-IT savvy person...

**Comment by rct** [comment]
Yes, you can run the PyScript kernel with the Home Assistant Community Add-On JupyterLab.

The steps are currently only documented in some discussions/issues.

See https://github.com/craigbarratt/hass-pyscript-jupyter/issues/32 

Note: the Jupyter PyScript kernel (the piece that makes the bridge between the PyScript integration running inside of Home Assistant and a Jupyter notebook is maintained in a separate repo https://github.com/craigbarratt/hass-pyscript-jupyter because it is (somewhat) of a separate component. 

Also you mentioned running on a RPi, the JupyterLab add-on uses a bit of memory. Depending on the amount of RAM in your RPi, you might only want to run JupyterLab when you are actively using it.
